src/main/java/net/datafaker/idnumbers/AlbanianIdNumber.java

```diff
@@ -11,13 +11,18 @@ public class AlbanianIdNumber implements IdNumbers {
     private static final String FIRST_CHAR = "0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZ";
     private static final String CHECKSUM_CHAR = "WABCDEFGHIJKLMNOPQRSTUV";
 
-    public String getInvalid(BaseProviders faker) {
-        String pin = getValid(faker);
+    @Override
+    public String country() {
+        return "AL";
+    }
+
+    public String generateInvalid(BaseProviders faker) {
+        String pin = generateValid(faker);
         int invalidMonth = faker.number().numberBetween(93, 99);
         return pin.substring(0, 2) + invalidMonth + pin.substring(4);
     }
 
-    public String getValid(BaseProviders faker) {
+    public String generateValid(BaseProviders faker) {
```

Comment:
This will break for anyone using it directly. We should deprecate for removal and it can call the new method.

Reply:
Also shouldn't generate valid have an override annotation?

Reply:
I guess my original comment here goes for any of these country classes

Reply:
Great!